.dev/docker/env/Dockerfile

```diff
@@ -36,44 +34,58 @@ RUN apt-get update \
       libusb-1.0-0-dev \
       libvtk${VTK_VERSION}-dev \
       libvtk${VTK_VERSION}-qt-dev \
+      lsb-release \
       qtbase5-dev \
       software-properties-common \
+      wget \
+      xvfb \
+ && if [ "$USE_LATEST_CMAKE" = true ] ; then \
+      apt-get -V install -y apt-transport-https ; \
+      wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null ; \
+      if [ "$(lsb_release -sr)" = "18.04" ]; then \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' ; \
+      else \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' ; \
+      fi \
+      && apt-get update ; \
+    fi \
+ && apt-get -V install -y cmake \
```

Comment:
And their keys as well?

Reply:
?

Reply:
> As an optional step, we recommend that you also install our kitware-archive-keyring package to ensure that your keyring stays up to date as we rotate our keys.

Reply:
Don't think we need this as we retrieve their keys everytime we build this image. This would be necessary when you update your packages later (e.g. take an Docker-image, run `apt-get update` and use the new layer), but as far as I know we don't do it(?)